MerlinAU.sh

```diff
@@ -771,7 +776,9 @@ Update_Custom_Settings()
         "FW_New_Update_ZIP_Directory_Path" | \
         "FW_New_Update_LOG_Directory_Path" | \
         "FW_New_Update_LOG_Preferred_Path" | \
-        "FW_New_Update_EMail_Notification")
+        "FW_New_Update_EMail_Notification" | \
```

Comment:
Approved.

Reply:
Unable to test at this time.

Reply:
Values default values found in file: 

FW_New_Update_Notification_Date TBD
FW_New_Update_Notification_Vers TBD
FW_New_Update_Postponement_Days="0"
FW_New_Update_EMail_Notification="true"

After updating.

Reply:
However:

FW_New_Update_EMail_CC_Name=TBD
FW_New_Update_EMail_CC_Address=TBD

Correctly shows up on a new settings file as expected. I only expect these settings to show up if CC is enabled :)

Reply:
Value is missing on update but exists on new creation. (For now)